models/cm_default.h

```diff
@@ -132,7 +143,7 @@ The following parameters can be set in the status dictionary.
  V_th       mV      Spike threshold (default: -55.0 mV)
 =========== ======= ===========================================================
 
-The following parameters can be set using the `AddCompartment` function
+The following parameters can be set using the ``AddCompartment`` function
```

Comment:
There is no ``AddCompartment`` function anymore. This is now done via the call to ``SetStatus()``.
```suggestion
The following parameters can be used when adding compartments using ``SetStatus()``
```